pangeo_forge_recipes/transforms.py

```diff
@@ -468,4 +468,4 @@ def expand(self, datasets: beam.PCollection) -> beam.PCollection:
         target_store = schema | PrepareZarrTarget(
             target=self.get_full_target(), target_chunks=target_chunks
         )
-        return rechunked_datasets | StoreDatasetFragments(target_store=target_store)
+        return rechunked_datasets | StoreDatasetFragments(target_store=target_store) | beam.create([target_store])
```

Comment:
```suggestion
        rechunked_datasets | StoreDatasetFragments(target_store=target_store)
        return target_store
```
@jbusecke, I think this would work?